Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -538,9 +540,18 @@ static void BeginField(FieldType field)
 			  "(VI->BeginField): Address: %.08X | WPL %u | STD %u | ACV %u | Field %s",
 			  xfbAddr, m_PictureConfiguration.WPL, m_PictureConfiguration.STD,
 			  m_VerticalTimingRegister.ACV, fieldTypeNames[field]);
-
+	
```

Comment:
Idk man, probably addd something then deleted it immediately and the whitespace ended up different